src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -160,6 +160,11 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
     } yield ()
   }
 
+  override def onGroupCreate(groupId: WorkbenchGroupIdentity): Future[Map[WorkbenchEmail, Seq[SyncReportItem]]] = {
```

Comment:
I don't think we want a generic onGroupCreate... feels like every group we create should call this which is not true (only managed groups)

Reply:
Resolved.  Had to add `synchronizeGroupMembers` to the `CloudExtensions` trait.